.gitignore

```diff
@@ -1 +1,3 @@
 .DS_Store
+*~.rb
```

Comment:
I'm assuming this is an editor-specific temp file. You might want to add this to your global gitignore.